cypress/e2e/brewery/ScenarioParameters-additional-advanced.cy.js

```diff
@@ -88,6 +94,8 @@ describe('Additional advanced scenario parameters tests', () => {
       .type('{moveToStart}' + '29/08/1997')
       .should('value', '29/08/1997');
 
+    BreweryParameters.selectScenarioToCompareOption(VALUES_TO_UPDATE.scenarioToCompare);
```

Comment:
We should probably add a check with `should('have.text', ...` before the _discard_ too